asv_bench/benchmarks/rolling.py

```diff
@@ -50,20 +50,24 @@ class Engine:
         ["int", "float"],
         [np.sum, lambda x: np.sum(x) + 5],
         ["cython", "numba"],
+        ["sum", "max", "min", "median", "mean"],
     )
-    param_names = ["constructor", "dtype", "function", "engine"]
+    param_names = ["constructor", "dtype", "function", "engine", "method"]
```

Comment:
are these benchmarks still reasonable in terms of total time, e.g. < 1s per

Reply:
Each benchmark is around ~1s. Is that too much?